src/examples/wasm/callback/wasm.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"strconv"
+	"syscall/js"
+)
+
+func main() {
+	document := js.Global().Get("document")
+	updateCallback := js.FuncOf(func(this js.Value, args []js.Value) interface{} {
+		update()
+		return nil
+	})
```

Comment:
I think it would be better if we could show in the example the use of the parameters and the return value. Anyway we can expand this to include that?

Reply:
That's a good idea, do you have an idea what would be a good example?

Reply:
Could we read the input from the parameters to this function instead of calling `getElementById`?

Reply:
What is this return value actually used for as well :thinking:?

Reply:
Callbacks can return something to JavaScript. However, I'm not sure why it is an `interface{}` and not a `js.Value`.